tests/strip-lines.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const assert = require('assert');
+const expect = require('chai').expect;
+const HTMLBarsMinifier = require('../lib/htmlbars-minifier');
+
+const minifier = new HTMLBarsMinifier('foo', {
+	stripIndentation: true,
+	removeTrailingSpaces: true,
```

Comment:
I think you should write these using the default values for everything but the flag you are testing, in this case `stripIndentation`. You already did it for the other tests